doc/api/fs.md

```diff
@@ -1524,11 +1524,27 @@ If `options` is a string, then it specifies the encoding. Example:
 ```js
 fs.readFile('/etc/passwd', 'utf8', callback);
 ```
+*Note*: When the path is a directory, the behavior of
+`fs.readFile()` and [`fs.readFileSync()`][] is platform-specific. On macOS,
+Linux, and Windows, an error will be returned. On FreeBSD, the contents
+of the files in the directory will be returned.
```

Comment:
Could you try it with  : `node -p "fs.readFileSync('test', 'utf-8')"`

I'm on MacOS, so I tried it with `fs.readFileSync()` on a file instead of a directory and it worked as expected!